spec/koans.js

```diff
@@ -8,31 +8,31 @@ describe('`let` restricts the scope of the variable to the current block - ', ()
   describe('`let` vs. `var`.', () => {
     it('`var` works as usual, it does not restricts scope', () => {
       if (true) {
-        /*You should add your code in here*/
+        var varX = true;
       }
-      // expect(varX).toBe(true);
+      expect(varX).toBe(true);
     });
 
     it('`let` restricts scope to inside the block', () => {
-      /*var or const? letX = false*/
       if (true) {
-        /*var or const? letX = true*/
+        var letX = false;
       }
-      //expect(letX).toBe(false);
+      expect(letX).toBe(false);
     });
 
     it('`var` does not restricts scope to inside the block in `for` loops', () => {
-      /*var or let? counter = 100*/
-      /*for (var or let? counter = 1; counter < 50; counter++){}*/
-
-      //expect(counter).toBe(50);
+      var counter = 100;
+      for (var counter = 1; counter < 50; counter++){
+        counter/2
+      }
+      expect(counter).toBe(50);
     });
```

Comment:
Ahaha! Malin mais ici le but cest de voir que var n'a pas un block scope, la variable counter est modifié dans la for loop, mais en dehors de cette loop elle garde al mémoire des changements effectués dans la loop (ça ne serait pas le cas avec let) : 
```js
    it('`var` does not restricts scope to inside the block in `for` loops', () => {
      var counter = 100;
      for (var counter = 1; counter < 50; counter++){}

      expect(counter).toBe(50);
    });
```